implementations/rust/ockam/ockam_command/src/markdown/mod.rs

```diff
@@ -169,8 +169,36 @@ fn generate_markdown_page(
         }
     }
 
-    // Subcommands list, if any
+    // Arguments and options or subcommands
     if cmd.get_subcommands().next().is_some() {
```

Comment:
This is my fault, but in the issue description I meant cmd.get_subcommands().next().**is_none()** here, so that:
- We display the arguments and options for commands that **don't** have subcommands
- We display only the subcommands if it does

Reply:
I've rewrote the description: https://github.com/build-trust/ockam/issues/5028